lib/detect-testing-library-utils.ts

```diff
@@ -242,11 +242,15 @@ export function detectTestingLibraryUtils<
      * then this method will return `true` ONLY IF a testing-library package
      * or custom module are imported.
      */
-    const isTestingLibraryImported: IsTestingLibraryImportedFn = () => {
+    const isTestingLibraryImported: IsTestingLibraryImportedFn = (
+      isStrict = false
+    ) => {
+      const isSomeModuleImported =
+        !!importedTestingLibraryNode || !!importedCustomModuleNode;
```

Comment:
```suggestion
        Boolean(importedTestingLibraryNode) || Boolean(importedCustomModuleNode);
```

Reply:
Why so picky with this detail?

Reply:
Whenever I see a double negation, I try to replace it by using the `Boolean` function, as that makes the intent a lot clearer to new people (especially for less experienced developers)

Reply:
I guess it's more _self-explanatory_, but I'd say it's not consistent with the rest of the codebase. I'd prefer tackling this sort of styling preferences in a separate PR, changing all related code to leave the codebase consistent in that sense (and even enabling some ESLint rule if necessary, like `no-implicit-coercion` for this case).

Reply:
A separate PR sounds good to me

Reply:
Are we happy to merge this then? 😅